### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by Kwaku28**
## This part of the project is implemented to add forms to the app

Tasks completed include the following;

1. Created a method called `current_user` in ApplicationController, which returns the first user from the database.
2. Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user`
    - Create a comment on behalf of the current_user
3. Users can add likes to Posts.

Thank you🤝

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,6 +6,11 @@
   get '/users', to: 'users#index'
   get '/users/:id', to: 'users#show'
   get '/users/:id/posts', to: 'posts#index'
-  get '/users/:id/posts/:id', to: 'posts#show'
+  get '/users/:id/posts/new', to: 'posts#new'
+  get '/users/:id/posts/:id', to: 'posts#show', as: 'post'
+  post '/users/:id/posts/create', to: 'posts#create'
+  get '/users/:id/posts/:post_id/comments/new', to: 'comments#new'
+  post '/users/:id/posts/:post_id/comments/create', to: 'comments#create'
+  post '/users/:id/posts/:post_id/likes', to: 'posts#likes', as: 'likes'
```
Comment (Shedrack-Sunday) [inline config/routes.rb]: - [ ] You're lacking a ``# controllers/likes_controller.rb`` controller to have the likes functionality. Kindly add this 👍 

### Instructions:
image

File: config/routes.rb [hunk 2]
```diff
@@ -6,6 +6,11 @@
...
```
Comment (Shedrack-Sunday) [inline config/routes.rb:15]: - [ ] Also, kindly add the route to add/create likes
### Sample code:
   ``POST   /posts/:post_id/likes(.:format)     likes#create``

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Kwaku28         👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with rails structured format✔️    
  
# Required Changes  🌵  

- [ ] You've done well, However, kindly check some key point note I pointed out to meet the requirements before moving to the next project 👍 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by cynthiainga — APPROVED** [review]
Hi @Kwaku28, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Implemented all changes requested ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**